src/python/WMCore/Services/pycurl_manager.py

```diff
@@ -165,6 +166,12 @@ def __init__(self, config=None, logger=None):
         self.followlocation = config.get('followlocation', defaultOpts['FOLLOWLOCATION'])
         self.maxredirs = config.get('maxredirs', defaultOpts['MAXREDIRS'])
         self.logger = logger if logger else logging.getLogger()
+        self.request_headers = None
```

Comment:
This seems not to be necessary. You might want to remove the TMP commented out code as well (from 100 lines below).

Reply:
done